File: VB/Controllers/HomeController.vb
Imports DevExpress.Web.Mvc
Imports System.Web.Mvc

Namespace ProjectTemplateNavigationMvc.Controllers
	Public Class HomeController
		Inherits Controller

		Public Function Index() As ActionResult
			Return View()
		End Function

		Public Function GetPartialView(ByVal itemName As String) As ActionResult
			Return RedirectToAction(itemName)
		End Function

		Public Function CallbackPanelPartial(ByVal itemName As String) As ActionResult
			ViewData("ActionName") = itemName
			Return PartialView()
		End Function

		Public Function View1Partial() As ActionResult
			Return PartialView()
		End Function

		Public Function View2Partial() As ActionResult
			Return PartialView()
		End Function

		Public Function View3Partial() As ActionResult
			Return PartialView()
		End Function

		Public Function View4Partial() As ActionResult
			Return PartialView()
		End Function
	End Class
End Namespace

File: CS/Controllers/HomeController.cs
using DevExpress.Web.Mvc;
using System.Web.Mvc;

namespace ProjectTemplateNavigationMvc.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return View();
        }

        public ActionResult GetPartialView(string itemName) {
            return RedirectToAction(itemName);
        }

        public ActionResult CallbackPanelPartial(string itemName) {
            ViewData["ActionName"] = itemName;
            return PartialView();
        }

        public ActionResult View1Partial() {
            return PartialView();
        }

        public ActionResult View2Partial() {
            return PartialView();
        }

        public ActionResult View3Partial() {
            return PartialView();
        }

        public ActionResult View4Partial() {
            return PartialView();
        }
    }
}